server/src/main/java/org/candlepin/resource/JobResource.java

```diff
@@ -174,6 +181,42 @@ public SchedulerStatus setSchedulerStatus(boolean running) {
         return getSchedulerStatus();
     }
 
+    @POST
+    @Path("cron/{task}")
```

Comment:
Would this make more sense (externally?) to call put this at jobs/schedule/:task?